deployment/openshift/about.md

```diff
@@ -1,12 +1,168 @@
+```
+
+Then, restart the Docker daemon:
+
+```console
+$ sudo systemctl restart docker
+```
+
+And let's run `oc cluster up` again:
+
+```console
+$ oc cluster up
+-- Checking OpenShift client ... OK
+-- Checking Docker client ... OK
+-- Checking Docker version ... OK
+-- Checking for existing OpenShift container ... OK
+-- Checking for openshift/origin:v1.3.1 image ... OK
+-- Checking Docker daemon configuration ... OK
+-- Checking for available ports ... OK
+-- Checking type of volume mount ...
+   Using nsenter mounter for OpenShift volumes
+-- Creating host directories ... OK
+-- Finding server IP ...
+   Using 10.0.2.15 as the server IP
+-- Starting OpenShift container ...
+   Creating initial OpenShift configuration
+   Starting OpenShift using container 'origin'
+   Waiting for API server to start listening
+   OpenShift server started
+-- Installing registry ... OK
+-- Installing router ... OK
+-- Importing image streams ... OK
+-- Importing templates ... OK
+-- Login to server ... OK
+-- Creating initial project "myproject" ... OK
+-- Server Information ...
+   OpenShift server started.
+   The server is accessible via web console at:
+       https://10.0.2.15:8443
+
+   You are logged in as:
+       User:     developer
+       Password: developer
+
+   To login as administrator:
+       oc login -u system:admin
+
+```
+
+Done! Follow the instructions on the screen to access the OpenShift Origin Web Console.
+
+## Deploying your first application
+
+You can deploy application from the Web Console or from the command line.
+As a demo and to test your new environment, you can deploy a sample Ruby application:
+
+```console
+$ oc new-app https://github.com/openshift/ruby-hello-world
+--> Found Docker image 8a8cd6f (43 hours old) from Docker Hub for "centos/ruby-22-centos7"
+
+    Ruby 2.2
+    --------
+    Platform for building and running Ruby 2.2 applications
+
+    Tags: builder, ruby, ruby22
+
+    * An image stream will be created as "ruby-22-centos7:latest" that will track the source image
+    * A Docker build using source code from https://github.com/openshift/ruby-hello-world will be created
+      * The resulting image will be pushed to image stream "ruby-hello-world:latest"
+      * Every time "ruby-22-centos7:latest" changes a new build will be triggered
+    * This image will be deployed in deployment config "ruby-hello-world"
+    * Port 8080 will be load balanced by service "ruby-hello-world"
+      * Other containers can access this service through the hostname "ruby-hello-world"
+
+--> Creating resources with label app=ruby-hello-world ...
+    imagestream "ruby-22-centos7" created
+    imagestream "ruby-hello-world" created
+    buildconfig "ruby-hello-world" created
+    deploymentconfig "ruby-hello-world" created
+    service "ruby-hello-world" created
+--> Success
+    Build scheduled, use 'oc logs -f bc/ruby-hello-world' to track its progress.
+    Run 'oc status' to view your app.
+```
```

Comment:
While this is nice, I wouldn't add it right here around these lines.

- It refers to OpenShift Online
- It talks about automated builds and build hooks that will not work with `oc cluster up` (because you normally will not have an externally accessible address for GitHub, etc)


We can, however, add this link to the references in the footer.